src/Chapter1.hs

```diff
@@ -537,7 +538,10 @@ value after "=" where the condition is true.
 Casual reminder about adding top-level type signatures for all functions :)
 -}
 
-mid x y z = error "mid: not implemented!"
+mid :: Int -> Int -> Int -> Int
+mid x y z = head $ filter (/= minimum xs) $ filter (/= maximum xs) xs
+    where
+        xs = [x, y, z]
```

Comment:
Wow! That is a clever solution 🧠 and very creative!

Going deeper into your idea, you could also do something like this:
```haskell
mid x y z = let (_:y:_) = sort [x, y, z] in y
```